pkg/services/ngalert/store/alert_rule.go

```diff
@@ -553,13 +551,11 @@ func (st DBstore) GetAlertRulesForScheduling(ctx context.Context, query *ngmodel
 			// In previous versions of Grafana, Loki datasources would default to range queries
 			// instead of instant queries, sometimes creating unnecessary load. This is only
 			// done for Grafana Cloud.
-			if lokiRangeToInstantEnabled {
-				if indices, migratable := canBeInstant(rule); migratable {
-					if err := migrateToInstant(rule, indices); err != nil {
-						st.Logger.Error("Could not migrate rule from range to instant query", "rule", rule.UID, "err", err)
-					} else {
-						st.Logger.Info("Migrated rule from range to instant query", "rule", rule.UID, "migrated_queries", len(indices))
-					}
+			if indices, migratable := canBeInstant(rule); migratable {
```

Comment:
Do we need not care to find out which ones can't be migratable?

Reply:
I agree with Josh here. Won't this will create a lot of noise at the Error level?

Is there ever a situation where an alert rule should have been translated into an instant query but couldn't? If so, can we log an error just for those cases?

Reply:
No we don't care about the ones that cannot be transformed. There is only a small subset that can, if the query type is `range` and if the aggregation is `last`.